IdentityCore/src/validation/MSIDAADAuthority.m

```diff
@@ -234,11 +236,6 @@ + (NSURL *)normalizedAuthorityUrl:(NSURL *)url
                           context:(id<MSIDRequestContext>)context
                             error:(NSError **)error
 {
-    if (![self isAuthorityFormatValid:url context:context error:error])
```

Comment:
Should we at least check that URL has at least 1 path component? Otherwise, this code will crash

Reply:
Sure. The reason behind it is the code path would never allow such a flow as url is validated before this method is called. 
However, in future, should we surface this API to be available outside of the class, it's good to have a safety net. Thus, I did for all authorities :)